code/obj/item/tank.dm

```diff
@@ -272,6 +229,44 @@ Contains:
 		else
 			..()
 
+/obj/item/tank/ui_interact(mob/user, datum/tgui/ui)
+	ui = tgui_process.try_update_ui(user, src, ui)
+	if (!ui)
+		ui = new(user, src, "GasTank", name)
+		ui.open()
+
+/obj/item/tank/ui_data(mob/user)
+	var/list/data = list()
+	data["pressure"] = MIXTURE_PRESSURE(air_contents)
+	data["maxPressure"] = PORTABLE_ATMOS_MAX_RELEASE_PRESSURE
```

Comment:
Old pressure bar used to regard the max release pressure of portable atmospherics as the "maximum pressure". I simply replicated this (because seeing the big green bar fill up all the way when you're refilling your tank feels really nice).